src/creditDecision/components/CreditDecisionTemplate.js

```diff
@@ -6,13 +6,13 @@ import CreditDecisionHistory from '$src/creditDecision/components/CreditDecision
 import {ContactTypes} from '$src/contacts/enums';
 
 type Props = {
-  businessId?: String,
-  contactId?: String,
-  contactType: String,
-  nin?: String,
+  businessId?: string,
```

Comment:
Is there anything else different between string and String but the looks? :thinking: 

I should propably then start to convert all the "capitalized" types also?